MainWindow.xaml.cs

```diff
@@ -612,19 +350,79 @@ private void EnsureDiscInformation()
             }
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        private Result EnsureCorrectInformationForSystemAndMediaType(KnownSystem? system, MediaType? type)
```

Comment:
Maybe rename to something like `GetSupportStatus`? The full info can go in the summary